prow/config.yaml

```diff
@@ -5388,6 +5388,49 @@ periodics:
       - name: cache-ssd
         hostPath:
           path: /mnt/disks/ssd0
+  - name: ci-kubernetes-e2e-kubeadm-gce-selfhosting
```

Comment:
@krzyzacy I think we may need to revisit how we rate-limit these jobs... 🤔 
Can we perhaps publish a `ci/latest-bazel` or something along those lines and seperate out these jobs instead of using `run_after_success` ?

Reply:
We could just have a special GCS path we point the bazel-build CI job at and have these CI jobs always fetch from there instead of waiting on the build job directly 🤔

Reply:
@BenTheElder Sounds reasonable! :+1: 
Once there is a decision here, I will modify the PR accordingly :)

Reply:
@xiangpengzhao thanks! I think we are probably OK to go forward with this PR, you just reminded me. (we recently had an issue with eating up the project pool https://github.com/kubernetes/test-infra/issues/5731https://github.com/kubernetes/test-infra/pull/5743)

@krzyzacy PTAL at https://github.com/kubernetes/test-infra/issues/5905 when get a chance